recipe/bld.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D DBUILD_SHARED_LIBS=OFF -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
```

Comment:
Boost 1.70 introduces "native" CMake config scripts which might be accidentally used.

```suggestion
cmake -G "NMake Makefiles" ^
      -DBoost_NO_BOOST_CMAKE=ON ^
      -D DBUILD_SHARED_LIBS=OFF ^
      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
      %SRC_DIR%
```